src/platform/haswell/include/platform/lib/memory.h

```diff
@@ -9,13 +9,16 @@
 
 #ifndef __PLATFORM_LIB_MEMORY_H__
 #define __PLATFORM_LIB_MEMORY_H__
+#include <config.h>
 
 #if !defined(__ASSEMBLER__) && !defined(LINKER)
 void platform_init_memmap(void);
 #endif
 
-/* physical DSP addresses */
+/* data cache line alignment */
+#define PLATFORM_DCACHE_ALIGN	sizeof(void *)
```

Comment:
Btw, core-isa.h in master for BDW/HSW is wrong, will be fixed soon, but please alsway use the macros from core-isa.h and not type sizes here. I would expect to see the macro for data cache line length here.

Reply:
It does not matter for this change and setting it directly to sizeof(void *) should be beneficial (update of core-isa won't cause problems with alignment).
And that is because as far as I know, we do not have L1 cache active on BDW/HSW.

Reply:
Fixed BDW ISA has
```
#define XCHAL_DCACHE_LINESIZE		8	/* D-cache line size in bytes */
```